src/map/script.c

```diff
@@ -8609,7 +8610,11 @@ BUILDIN(readparam) {
 	}
 
 	if (script_hasdata(st, 3)) {
-		sd = script->nick2sd(st, script_getstr(st, 3));
+		if (script_isstringtype(st, 3)) {
+			sd = script->nick2sd(st, script_getstr(st, 3));
+		} else {
+			sd = script->id2sd(st, script_getnum(st, 3));
+		}
```

Comment:
unless I'm missing something, but the [coding guideline says this](https://github.com/HerculesWS/Hercules/wiki/Coding-Style#chapter-3-placing-braces-and-spaces)

Do not unnecessarily use braces where a single statement will do.
```
	if (condition)
		action();
```

and
```
	if (condition)
		do_this();
	else
		do_that();
```

Reply:
seems this PR kinda outta my hands now lol yea this one i guess personal style vs our guidelines.